content/22.vaccines.md

```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
+
+There are at least five COVID-19 LAV candidates at various stages of vaccine development.
+A single-dose LAV candidate referred to as YF-SO used live-attenuated yellow fever 17D (YF17D) as a vector for a noncleavable prefusion conformation of the SARS-CoV-2 antigen.
+This LAV has been assessed in hamsters, mice, and macaques [@doi:10.1038/s41586-020-3035-9].
+YF-SO induced a robust immune response in all three animal models and prevented SARS-CoV-2 infection in macaques and hamsters [@doi:10.1038/s41586-020-3035-9].
+Other LAVs being investigated for the prophylaxis of COVID-19 include a Bacillus Calmette-Guerin (BCG) vaccine sponsored by institutes in Australia in collaboration with the Bill and Melinda Gates Foundation, which is in phase III clinical testing [@clinicaltrials:NCT04327206]. 
+A second investigation using a BCG vaccine is also ongoing, which is led by Texas A&M University in collaboration with numerous other U.S. institutions [@clinicaltrials:NCT04348370].
+The purpose of the BCG vaccine is to prevent tuberculosis, however, it is known to exert protective non-specific effects against other respiratory tract infections in _in vitro_ and _in vivo_ studies [@doi:10.1016/j.cmi.2019.04.020], hence the interest many have for its potential use against COVID-19 [@doi:10.1038/s41577-020-0337-y].
+
+In april 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/], however, there has been no updates on the findings of their preclinical testing. 
+Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
+Following successful preclinical investigation [@doi:10.1073/pnas.2102775118], an intranasally administered deoptimized SARS-CoV-2 LAV known as COVI-VAC was developed by both New York-based Codagenix and the Serum Institute of India. 
+COVI-VAC entered phase I human trials and dosed its first participants in January 2021 [04619628][@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].  
+It is anticipated that the COVI-VAC phase I human trials will be complete by May 2022.
+Similarly,  Meissa vaccines in Kansas in the U.S., who also develop vaccines for respiratory syncytial virus (RSV), began enrollment for its phase I human trials on its intranasal live-attenuated chimeric vaccine candidate in March 2021, but they are still recruiting participants [@clinicaltrials:NCT04798001][@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
```

Comment:
```suggestion
Similarly,  Meissa Vaccines in Kansas, U.S.A., which also develops vaccines for _Respiratory syncytial virus_ (RSV), began enrollment for phase I human trials on an intranasal live-attenuated chimeric vaccine candidate in March 2021 but is still recruiting participants [@clinicaltrials:NCT04798001; @url:https://covid-19tracker.milkeninstitute.org].
```
It switches back and forth between treating Meissa vaccines as "it" and "they" -- does this work?

Reply:
maybe "...but they are still recruiting..." tricky one

Reply:
Maybe drop the subject entirely? 😆 
```suggestion
Similarly,  Meissa Vaccines in Kansas, U.S.A., which also develops vaccines for _Respiratory syncytial virus_ (RSV), began enrollment for phase I human trials on an intranasal live-attenuated chimeric vaccine candidate in March 2021 for which recruitment is ongoing [@clinicaltrials:NCT04798001; @url:https://covid-19tracker.milkeninstitute.org].
```